vllm/engine/arg_utils.py

```diff
@@ -838,7 +838,9 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             **parallel_kwargs["data_parallel_external_lb"],
         )
         parallel_group.add_argument(
-            "--enable-expert-parallel", **parallel_kwargs["enable_expert_parallel"]
+            "--enable-expert-parallel",
+            "-ep",
```

Comment:
It feels a bit confusing.

I have `-tp 8` and `-dp 8`, and `-ep` might be mistakenly interpreted as also needing to be set to `-ep 8`.